### PR TITLE
Wip/goldbach beta release

### LINKED ISSUE #727: What is example.ttl? Should it exist?
I cannot find the file. Why should it be loaded?

https://github.com/dhlab-basel/Knora/blob/f804346ed457f69ae3a65ab0b4a3da4c8d60feb7/webapi/scripts/graphdb-knora-test-data.expect#L49

### PR DESCRIPTION
**Pull request by tobiasschweizer**
changes needed for the beol beta release

closes #727

### CODE REVIEW

File: webapi/scripts/graphdb-knora-test-data.expect [hunk 1]
```diff
@@ -38,20 +38,23 @@ send "load ../_test_data/ontologies/anything-onto.ttl into http://www.knora.org/
...
-send "load ../_test_data/all_data/beol-data.ttl into http://www.knora.org/data/beol .\r"
+send "load ../_test_data/all_data/beol-data.ttl into http://www.knora.org/data/0801/beol .\r"
 expect $prompt
+
+
 send "load ../_test_data/ontologies/biblio-onto.ttl into http://www.knora.org/ontology/biblio .\r"
 expect $prompt
 send "load ../_test_data/all_data/biblio-data.ttl into http://www.knora.org/data/biblio .\r"
 expect $prompt
-send "load ../_test_data/ontologies/example.ttl into http://www.knora.org/ontology/0001/example .\r"
```
Comment (tobiasschweizer) [inline webapi/scripts/graphdb-knora-test-data.expect]: see #727

**Comment by tobiasschweizer** [comment]
@SepidehAlassi @benjamingeer Can this PR be merged?

File: webapi/_test_data/ontologies/beol-onto.ttl [hunk 2]
```diff
@@ -922,6 +1465,16 @@
                             owl:minCardinality "0"^^xsd:nonNegativeInteger ;
```
Comment (tobiasschweizer) [inline webapi/_test_data/ontologies/beol-onto.ttl]: Shouldn't this be `owl:maxCardinality "1"^^xsd:nonNegativeInteger`? Or may a letter have more than one translation?

**Comment by benjamingeer** [comment]
> Can this PR be merged?

You're probably better able to answer that question than me. :)

**Comment by tobiasschweizer** [comment]
> You're probably better able to answer that question than me. :)

You approved it ;-) 

I will see if Sepideh is ok with merging it.

**Comment by benjamingeer** [comment]
> You approved it ;-)

Yes exactly, if I didn't think it could be merged, I wouldn't have approved it. But if I missed anything, you're more likely to notice than me.

**Comment by tobiasschweizer** [comment]
> Yes exactly, if I didn't think it could be merged, I wouldn't have approved it. But if I missed anything, you're more likely to notice than me.

Dear Ben,

I get it, I just wanted to be polite and not interfere by just merging it myself since I was on a holiday and then sick for a few days (and thus probably not aware of all the recent developments).

**Comment by benjamingeer** [comment]
> and thus probably not aware of all the recent developments

OK I understand. After I approved the PR, Sepideh said she needed to make a couple of small changes. I said if they were small, she could go ahead and merge anyway.

**Comment by tobiasschweizer** [comment]
Ok, I will see what she says :-)

Have a nice day!

File: webapi/_test_data/ontologies/beol-onto.ttl (hunk 2 above)
Comment (SepidehAlassi) [reply webapi/_test_data/ontologies/beol-onto.ttl]: a letter may have more than one translation, it is not the case for Goldbach volume, but might be the case for other leoo volumes

Comment (tobiasschweizer) [reply webapi/_test_data/ontologies/beol-onto.ttl]: Ok, then this is fine :-)

**Pull request merged**